java/test/jmri/jmrit/logix/BlockOrderTest.java

```diff
@@ -1,30 +1,91 @@
 package jmri.jmrit.logix;
 
+import jmri.InstanceManager;
+import jmri.SignalHead;
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
 import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
-import org.junit.Ignore;
-import org.junit.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  *
  * @author Paul Bender Copyright (C) 2017	
  */
-public class BlockOrderTest {
```

Comment:
We are in the middle of a long running process of migrating from JUnit 3 to JUnit 4 tests. You might not want to not migrate this test from 4 to 3.